src/events/SDL_pen.c

```diff
@@ -492,6 +553,13 @@ void SDL_SendPenButton(Uint64 timestamp, SDL_PenID instance_id, const SDL_Window
             event.pbutton.button = button;
             event.pbutton.down = down;
             SDL_PushEvent(&event);
+
+            if (window && (pen_touching == instance_id)) {
```

Comment:
Check mouse->pen_mouse_events?

Reply:
Good catch, fixed in 82d97706d6e78d10d17dd81441d5af9cf09aff25.